packages/web-components/src/dialog/dialog.stories.ts

```diff
@@ -0,0 +1,239 @@
+import { html } from '@microsoft/fast-element';
+import type { Args, Meta } from '@storybook/html';
+import { renderComponent } from '../helpers.stories.js';
+import type { Dialog as FluentDialog } from './dialog.js';
+import './define.js';
+import '../button/define.js';
+import '../text/define.js';
+
+type DialogStoryArgs = Args & FluentDialog;
+type DialogStoryMeta = Meta<DialogStoryArgs>;
+
+const DismissCircle24Regular = html`
+  <svg
+    fill="currentColor"
+    aria-hidden="true"
+    width="24"
+    height="24"
+    viewBox="0 0 24 24"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <path
+      d="M12 2a10 10 0 1 1 0 20 10 10 0 0 1 0-20Zm0 1.5a8.5 8.5 0 1 0 0 17 8.5 8.5 0 0 0 0-17Zm3.45 4.9.08.07c.27.27.3.68.07.98l-.07.08L13.06 12l2.47 2.47c.27.27.3.68.07.98l-.07.08a.75.75 0 0 1-.98.07l-.08-.07L12 13.06l-2.47 2.47a.75.75 0 0 1-.98.07l-.08-.07a.75.75 0 0 1-.07-.98l.07-.08L10.94 12 8.47 9.53a.75.75 0 0 1-.07-.98l.07-.08a.75.75 0 0 1 .98-.07l.08.07L12 10.94l2.47-2.47a.75.75 0 0 1 .98-.07Z"
+      fill="currentColor"
+    ></path>
+  </svg>
+`;
+
+const closeDialog = (e: Event, id: string) => {
+  const dialog = document.getElementById(id) as FluentDialog;
+  dialog.hide();
+};
+
+const openDialog = (e: Event, id: string) => {
+  const dialog = document.getElementById(id) as FluentDialog;
+  dialog.show();
+};
+
+const dialogTemplate = html<DialogStoryArgs>`
+  <style>
+    ::part(positioning-region) {
+      z-index: 9999;
+    }
+  </style>
+  <div>
+      <fluent-button @click=${(e: Event, c) => openDialog(e, 'dialog-default')}>Open Dialog</fluent-button>
+      <fluent-dialog 
+        id="dialog-default" 
+        ?modal=${x => x.modal} 
+        ?alert=${x => x.alert} 
+        ?no-trap-focus=${x => x.noFocusTrap} 
+        hidden
+      >
+        <div slot="title">Dialog Title</div>
+        <div><code>Dialog</code> is a window overlaid on either the primary window or another dialog window. Windows under a modal dialog are inert. That is, users cannot interact with content outside an active dialog window. Inert content outside an active dialog is typically visually obscured or dimmed so it is difficult to discern, and in some implementations, attempts to interact with the inert content cause the dialog to close.</div>
+        <fluent-button 
+          slot="actions" 
+          appearance="primary" 
+          @click="${(e: Event, c) => closeDialog(e, 'dialog-default')}">Close Drawer</fluent-button>
+        <fluent-button
+          slot="actions">Do Something</fluent-button>
+      </fluent-dialog>
+    </div>
+  </div>
+`;
+
+export default {
+  title: 'Components/Dialog',
+  args: {
+    modal: false,
+    alert: false,
+    hidden: false,
+    noTrapFocus: false,
+  },
+  argTypes: {
+    hidden: {
+      description: 'Renders dialog as a modal',
```

Comment:
I think the description should read, "Sets the visibility of the dialog" from your spec.